# HG changeset patch
# User Andrew Janke <andrew@apjanke.net>
# Date 1530547468 25200
#      Mon Jul 02 09:04:28 2018 -0700
# Node ID 975450679c15510c25dd86b8ad606a3b117dddcb
# Parent  075b9f8a32d8d7cf8fa3a5cd406f89dcca115b1a
.hgignore: Add more files to ignore (bug #54216).

* .hgignore: Ignore HG-ID.  Ignore intermediate compilations *.Plo, *.Po.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -30,24 +30,29 @@ syntax: regexp
 ^build-aux/ltmain\.sh$
 ^build-aux/mdate-sh$
 ^build-aux/missing$
 ^build-aux/move-if-change$
 ^build-aux/texinfo\.tex$
 ^build-aux/ylwrap$
 ^config\.in\.h$
 ^configure$
+^HG-ID$
 
 # e.g. doc/faq/OctaveFAQ.info
 #      doc/interpreter/octave.info-4
 ^doc/\w+/\w+\.info(-\d+)?$
 
 ^doc/\w+/stamp-(vti|\d+)$
 ^doc/\w+/version-\w+\.texi$
 ^doc/interpreter/images\.mk$
 
 # e.g. liboctave/operators/smx-op-inc.mk
 ^liboctave/operators/\w+-op-\w+\.mk$
 
 ^libinterp/dldfcn/module\.mk$
 
 # DOCSTRINGS files are built in the source tree
 ^(libinterp|scripts)/DOCSTRINGS$
+
+# Intermediate compilation results for libraries
+.*\.Plo
+.*\.Po
